File: src/LC_242.java
public class LC_242 {
  public boolean isAnagram(String s, String t) {
    if (s.length() != t.length()) return false;
    int[] counter = new int[26];
    for (char element : s.toCharArray()) {
      counter[element - 'a']++;
    }
    for (char element : t.toCharArray()) {
      counter[element - 'a']--;
    }
    for (int count : counter) {
      if (count != 0) return false;
    }    
    return true;
  }
}

File: src/LC_242.js
var isAnagram = function(s, t) {
  if (s.length !== t.length) return false;
  let counter = Array.from({length: 26}, () => 0);
  for (let char of s) {
    counter[char.charCodeAt(0) - 'a'.charCodeAt(0)]++;
  }
  for (let char of t) {
    counter[char.charCodeAt(0) - 'a'.charCodeAt(0)]--;
  }
  for (let count of counter) {
    if (count !== 0) return false;
  }
  return true;
};